worlds/hylics2/Options.py

```diff
@@ -37,6 +49,6 @@ class Hylics2Options(PerGameCommonOptions):
     party_shuffle: PartyShuffle
     gesture_shuffle: GestureShuffle
     medallion_shuffle: MedallionShuffle
-    random_start: RandomStart
+    start_location: StartLocation
     extra_items_in_logic: ExtraLogic
     death_link: Hylics2DeathLink
```

Comment:
While you're in here, might as well add a newline to the end of file
```suggestion
    death_link: Hylics2DeathLink
    
```